.github/workflows/main.yml

```diff
@@ -2,6 +2,10 @@ name: Run Konveyor main branch tests
 
 on: ["push", "pull_request"]
 
+concurrency:
+  group: Konveyor-main-branch-tests-${{ github.ref }}
```

Comment:
I'm not sure this is a problem, but the workflows are executed for `main` as well as `release-XY` branches and I don't think we don't want cancel workflows across branches.

Reply:
It won't be a problem.  This is exactly how I have the groups setup on tackle2-ui so that multiple pushes to main, or updates to a PR, can stop a ~35min long e2e test that is already invalid before it finishes running.

Here is where I picked up the trick (the second example under the heading): https://docs.github.com/en/actions/using-jobs/using-concurrency#example-concurrency-groups